src/inc/corprof.idl

```diff
@@ -632,12 +632,14 @@ typedef enum
     // Enables the MovedReferences/MovedReferences2 callback for compacting GCs only.
     COR_PRF_HIGH_MONITOR_GC_MOVED_OBJECTS           = 0x00000020,
 
-    COR_PRF_HIGH_REQUIRE_PROFILE_IMAGE              = 0,
```

Comment:
@noahfalk `COR_PRF_HIGH_REQUIRE_PROFILE_IMAGE` has been here since the initial commit for coreclr, but I can't imagine a use for a flag without a value. Was it just a typo?

Reply:
hmm I guess I should fix it even though your comment makes sense.

Reply:
Removing the flag is technically a breaking change, but since we don't consume that flag anywhere internally in the CLR it would only potentially cause compile issues when people upgraded their corprof.h to the latest.

I am for taking it out since it appears to be pointless, I just wanted to check with Noah in case I'm missing historical context.

Reply:
It may have been a typo, but I'd suggest leaving it so we don't cause some profiler to have a compile error just in case. I don't think it does any harm other than adding a little noise.